data/hai/unfettered 0 prologue.txt

```diff
@@ -506,57 +509,1315 @@ mission "Unfettered Jump Drive Trading"
+			`	Many pilots seem to have taken up the call, but it appears the organizers here are still interested in all the volunteers they can get. Now that you're here, you're not sure you'll be able to leave without at least being accosted for your assistance with this.`
+			choice
+				`	(Signal them that I'm going to get the supplies.)`
+					to display
+						has "language: Wanderer"
+					goto agree
+				`	(Ask a local ship to translate.)`
+					to display
+						not "language: Wanderer"
+					goto translation
+				`	(I don't have time for this.)`
+			label leave
+			`	You manage to leave without anyone stopping you in all the rush, though you're sure someone was trying to get your attention. If they remember you, the Unfettered will likely not be too happy. You probably also shouldn't stay landed here too long in case they decide to follow you.`
+				decline
+			label translation
+			`	You hail the nearest Unfettered ship, sending a request for a translation of the message. Surprisingly, they are more than willing to translate the important parts for you, although they have trouble finding some of the words. "There is a... 'plague' here. Go to <stopovers> to get the.. the anti-plague! Now!" They then cut the transmission.`
+			choice
+				`	(Signal them that I'm going to get the supplies.)`
+				`	(I don't have time for this.)`
+					goto leave
+			label agree
+			`	You are quickly assigned the collection of supplies from <stopovers> and urged to return them as quickly as possible.`
+				accept
+	on stopover
+		dialog `You're surprised to see the Hai are more than happy to give you the medical supplies needed. They load up the cargo and, once they are done, one of them addresses you. "Tell our brothers and sisters to stay strong. Hopefully it is not too late," he says with determination.`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog phrase "generic missing stopover or cargo"
+	on complete
+		log `Brought medial supplies to Darkcloak to prevent an epidemic.`
+		log "People" "Sookootow" `An Unfettered Chief worried for his people, inside Hai or Unfettered territory. He seems to believe that all Hai are equal, and as a member of the council he has significant influence.`
+		payment 1500000
+		conversation
+			`The staff seem greatly relieved when you return promptly with the medical supplies. As you're returning to your ship, one of them catches up to you. "Hello, captain <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally. I find it quite admirable that an outsider would risk getting caught up in all of this. It would mean a great deal to me, if your bravery holds fast, so make sure to check the job board for more jobs like this."`
```

Comment:
```suggestion
			`The staff seem greatly relieved when you return promptly with the medical supplies. As you're returning to your ship, one of them catches up to you. "Hello, captain <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally. I find it quite admirable that an outsider would risk getting caught up in all of this. It would mean a great deal to me if your bravery holds fast, so make sure to check the job board for more jobs like this."`
```